apps/governance/.env.mainnet

```diff
@@ -5,6 +5,7 @@ NX_VEGA_URL=https://api.vega.community/graphql
 NX_VEGA_NETWORKS='{"DEVNET":"https://dev.governance.vega.xyz","TESTNET":"https://governance.fairground.wtf","MAINNET":"https://governance.vega.xyz"}'
 NX_ETHEREUM_PROVIDER_URL=https://mainnet.infura.io/v3/3ba145ccc2884bcd91213d8dc989ca76
 NX_ETHERSCAN_URL=https://etherscan.io
+NX_ARBISCAN_URL=https://arbiscan.io
```

Comment:
We already have two places defining an arbiscan root url. Now this adds a third. Probably not something to fix right now but I'd like to unify this in a .ts file which can be indexed into with a chain id

Reply:
Based on the fact that the EtherscanLink component already handles other chains we don't need this new ENV var at all